backend/handlers/handlers.go

```diff
@@ -1068,6 +1072,34 @@ func (handlers *Handlers) getExchangeMoonpayBuy(r *http.Request) (interface{}, e
 		return nil, err
 	}
 
+	return acct, nil
+}
+
+func (handlers *Handlers) getExchangePocketBuy(r *http.Request) (interface{}, error) {
+	acct, err := handlers.getAccountFromCode(mux.Vars(r)["code"])
+	if err != nil {
+		return nil, err
+	}
+
+	return exchanges.BuyPocket(acct)
```

Comment:
fyi, errors returned like this are swallowed in Qt (we should refactor error handling in the handlers). if you want to handle errors, for now we normally return a struct with a success flag and error message.

Reply:
Fixed